### PR TITLE
added the Profile / Company Details / Password sections of the Settin…

### PR DESCRIPTION
**Pull request by Rahim-444**
…gs page

### CODE REVIEW

File: app/(root)/layout.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import './globals.css';
 
 import { Analytics } from '@vercel/analytics/react';
-import { SessionProvider } from 'next-auth/react';
-import { SpeedInsights } from "@vercel/speed-insights/next"
+import { SessionProvider } from '@/components/component/authProvider';
```
Comment (keyskull) [inline app/(root)/layout.tsx]: Weird operation

File: components/component/authProvider.tsx
Comment (keyskull) [inline components/component/authProvider.tsx:1]: You can't access `next-auth/react` package directly?

File: package.json
Comment (keyskull) [inline package.json:1]: no need to update the `package.json`

**Review by keyskull — CHANGES_REQUESTED** [review]

File: app/(root)/layout.tsx (hunk 1 above)
Comment (Rahim-444) [reply app/(root)/layout.tsx]: you have to make it a client component for more context checkout https://stackoverflow.com/questions/75902311/next-js-13-and-next-auth-issues-with-usesession-and-sessionprovider

**Pull request closed without merging**